twilight-model/src/guild/onboarding/prompt.rs

```diff
@@ -0,0 +1,71 @@
+use crate::id::{marker::OnboardingPromptMarker, Id};
+use serde::{Deserialize, Serialize};
+
+use super::{OnboardingPromptOption, OnboardingPromptType};
+
+/// A prompt in the onboarding flow.
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct OnboardingPrompt {
```

Comment:
Remove initial "The" and "A" prefixes, `s/id/ID`, and end each sentence with a dot.